docs/DeveloperGuide.adoc

```diff
@@ -721,17 +721,76 @@ Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (un
 [width="59%",cols="22%,<23%,<25%,<30%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
-|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
-
 |`* * *` |user |add a new person |
 
 |`* * *` |user |delete a person |remove entries that I no longer need
 
+|`* * *` |user |request for a help guide |know more about the address book and use it with less difficulty
+
+|`* * *` |user |list and view all my contacts |
+
+|`* * *` |user |edit a person's information |constantly update or correct a contact's information
+
+|`* * *` |user |select a person by index |use the information provided to search on Google
+
+|`* * *` |user |view the history of the commands inputted |
+
+|`* * *` |user |undo the commands that were inputted |revert unnecessary or unwanted changes
+
+|`* * *` |user |redo the commands that is undone |revert changes that were undone
+
+|`* * *` |user |clear entries in my addressbook |
+
+|`* * *` |user |exit the program |
+
 |`* * *` |user |find a person by name |locate details of persons without having to go through the entire list
 
+|`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
+
+|`* * *` |user |have a CLI prompt that can exhibit the required information to add an individual  |avoid inputting the wrong information
+
+|`* * *` |user |store multiple phone numbers and email addresses for any contact |store extra communication information of my contact
+
+|`* * *` |user |access Facebook, LinkedIn and Google |search and add my contacts on respective websites
+
+|`* * *` |user |have a confirmation request of the details inputted before adding an individual|make changes to any errors inputted
+
+|`* * *` |user |have a confirmation request & warning message upon using the clear command |avoid accidentally clearing my address book
+
+|`* * *` |user |find contacts based on part of their names |
```

Comment:
so that ... ?

Reply:
Just updated the purpose of certain user stories and also added some specific user types